code/modules/power/singularity/singularity.dm

```diff
@@ -167,7 +167,7 @@
 	dissipate(seconds_per_tick)
 	radiation_pulse(src, 4, intensity = min(5000, (energy * 4.5) + 1000), should_rad_act = FALSE)
 	for(var/obj/collector in range(5, src))
-		if(!istype(collector, /obj/machinery/power/rad_collector))
+		if(!istype(collector, /obj/machinery/power/rad_collector) && !istype(collector, /obj/structure/artifact) && !istype(collector, /obj/item/melee/artifact) && !istype(collector, /obj/item/gun/magic/artifact) && !istype(collector, /obj/item/stock_parts/cell/artifact))
```

Comment:
I would recommend turning these bulk istype() checks into a single isartifact() macro define, might also want macros for each individual artifact type as well.